client/src/components/SessionForm.jsx

```diff
@@ -28,38 +31,74 @@ const SessionForm = ({ state }) => {
     const [successMessage, setSuccessMessage] = useState(false);
 
     //Handle submit function to POST session details to server/database
-    const handleSubmit = (event) => {
+    const handleSubmit = async (event) => {
         event.preventDefault();
         
         //Print out what would be the submission
         console.log("We'd be submitting now");
         console.log(inputs);
```

Comment:
Are the console.log for debugging/ confirming the functionality?

Reply:
Yes exactly right for those two! I initially had it in as I was confirming that the state was holding the right information, but then I decided to leave it in the code once I got it working to use during demos.